allegro/src/bitmap.rs

```diff
@@ -21,6 +21,21 @@ pub struct Bitmap
 
 impl Bitmap
 {
+
+	/// This method is used to create a new Bitmap object.
+	/// Can be used to create a bitmap of w width and h height.
+	/// 
+	/// # Examples
+	///  ```
+	/// use allegro::*;
+	/// 
+	/// pub fn blank_bitmap(core: &Core) {
```

Comment:
"blank" implies something like black to me, but new bitmaps have undefined contents. I'm not sure this function is actually that useful. If you think this needs an example, then something that does: new bitmap + set target + clear to color would be more useful.

Reply:
"this function is actually that useful"
I usually draw to a blank bitmap then pass the pointer to that bitmap and use al_draw_bitmap(*bmp, 0, 0, 0);

I'm also not sure these documentation submissions are desired which is why I wanted to test the waters for now by submitting a few pr's. If you don't think they are needed or useful in general let me know.  My feelings won't be hurt

Just to clarify. My goal of with the examples is to create a minimum viable example (At least for now) since I'm a beginner. If you would rather someone with more experience with your library do the examples that's fine. I just thought I'd try to do something (Maybe) useful while I work through it. 

I'll just keep learning the library for now either way. I'm pretty new to rust and just seeing a minimum example is useful to me. Especially coming from a allegro C user to using this wrapper.

I might make another push with the changes you suggest if you think this will be useful.

Reply:
I think it's okay if some functions don't have examples. I think my personal philosphy would be to have a quick example for the entire class in the class's comment, but for individual functions I'd only add them when it's appropriate.

Reply:
Understood. I'll consider that when submitting future PR's to documentation comments on a class level. 
Feel free to close or cherry pick anything from this PR that you think would be helpful. 
I "thought" this would have been helpful, but I'm new to Rust so having things spelled out to me is helpful.

I may come back later after spending more time with the language with some more insightful documentation.

Reply:
I think even this is pretty good, thank you. Tell me if you think you're done with it and I can merge it.

Reply:
That's good. I pushed a new commit to the branch I'm trying to merge in Battojutsu:jutsu/documentation

I appreciate the code review. It was insightful. I'll submit some more documentation updates later now that I see you are open to them. I'll be sure to try to keep them in separate commits so you can cherry-pick what you think is needed.